components/src/core/Hero/Hero.tsx

```diff
@@ -117,20 +120,24 @@ const HeroRender: React.ForwardRefRenderFunction<unknown, HeroProps> = (props: H
             ref={ref}
             className={cx(defaultClasses.root, classes.root, userClassName)}
             data-test={'wrapper'}
-            iconSize={iconSize}
-            iconBackgroundColor={iconBackgroundColor}
             {...otherProps}
         >
-            <span className={cx(defaultClasses.icon, classes.icon)}>{icon}</span>
-            <span className={cx(defaultClasses.values, classes.values)}>
+            <Icon
+                className={cx(defaultClasses.icon, classes.icon)}
+                iconSize={iconSize}
+                iconBackgroundColor={iconBackgroundColor}
+            >
+                {icon}
+            </Icon>
+            <Values  component={'span'} className={cx(defaultClasses.values, classes.values)}>
                 {!props.children && ChannelValueProps?.value && (
                     <ChannelValue fontSize={ChannelValueProps?.fontSize} {...ChannelValueProps} />
                 )}
                 {props.children}
-            </span>
-            <Typography variant={'body1'} color={'inherit'} className={cx(defaultClasses.label, classes.label)}>
+            </Values>
+            <Label variant={'body1'}  component={'span'} color={'inherit'} className={cx(defaultClasses.label, classes.label)}>
```

Comment:
As we are passing component={'span'} to Label component, we need to give type to the styled Label component. Typography does not accept component prop, so we need to give <TypographyProps & BoxProps>.

Reply:
I've removed the `component={span}` since Label is a Typography & not a Box.